experimental/packages/opentelemetry-exporter-prometheus/src/PrometheusSerializer.ts

```diff
@@ -33,10 +33,7 @@ function escapeString(str: string) {
   return str.replace(/\\/g, '\\\\').replace(/\n/g, '\\n');
 }
 
-function escapeLabelValue(str: string) {
-  if (typeof str !== 'string') {
```

Comment:
Probably because it was previously possible to have different types. In the original metrics implementation, labels could have any type. Then they were restricted to string specifically, and now they can have any type again.

Moreover, if someone is using javascript not typescript they may add invalid attributes so we tend to be pretty defensive with attributes/labels provided by user.